rviz_default_plugins/src/rviz_default_plugins/tools/nav_goal/goal_tool.cpp

```diff
@@ -66,7 +66,7 @@ void GoalTool::updateTopic()
 {
   // TODO(anhosi, wjwwood): replace with abstraction for publishers once available
   publisher_ = context_->getRosNodeAbstraction().lock()->get_raw_node()->
-    template create_publisher<geometry_msgs::msg::PoseStamped>(topic_property_->getStdString());
+    template create_publisher<geometry_msgs::msg::PoseStamped>(topic_property_->getStdString(), 10);
```

Comment:
In these cases an explicit local variable for the numeric parameter might be more readable, like https://github.com/ros2/rosidl_python/blob/c66c3b7075a8e9c9ef01826f547d7996e0c6c604/rosidl_generator_py/resource/_msg_support.c.em#L623-L624

This certainly doesn't need to be addressed to get this merged - it is merely a thought.